e2e/cypress/e2e/designer/notifyOutput.js

```diff
@@ -0,0 +1,39 @@
+import { When, Then } from "@badeball/cypress-cucumber-preprocessor";
+
+When("I open the lists menu", () => {
+  cy.findByTestId("menu-lists").click();
+});
+
+When("Add a new list with 3 list items", (table) => {
```

Comment:
seeing as you're testing outputs, and not lists/list items, ideally you don't want to create a list here. 

1. create a form json with these list items
2. save it to fixtures/notifyOutput.json
3. use `Given the form "notifyOutput" exists` 
  - this loads the json, and makes a post request to the runner
  - then visits the form in the designer
  - you should see your list items are there

Reply:
Given - sets up the starting state of your test. we're testing that notify output shows list items here, not if the designer can create list items. Although they run pretty fast locally, they will be slower on CI, so the extra few steps will "hurt".  

When - some actions a user/process does

Then - Your test condition